.circleci/config.yml

```diff
@@ -254,13 +254,13 @@ jobs:
       - checkout
       - restore_cache:
           keys:
-            - v2-deps-{{ checksum "ui/yarn.lock" }}
-            - v2-deps-
+            - v3-deps-{{ checksum "ui/yarn.lock" }}
+            - v3-deps-
```

Comment:
My guess is that the failures was due to a corrupt/poisoned cache that got loaded in the fallback condition.  I'd recommend removing this fallback, and ensuring that we always use a clean dependencies installation whenever we change package.json.  Installing dependencies take ~3 minutes and we seem do it rarely (once every few months?!), so the overhead won't be much hopefully.  We can add it back if it proves to be a problem

```suggestion
```